### PR TITLE
create widgets training notebook

### PR DESCRIPTION
**Pull request by haileyajohnson**
Address #44 
- Create a Widgets training notebook
- (temporarily removed) add links to notebook on project page and resources page

### CODE REVIEW

File: notebooks/notebook_enhancements/NotebookWidgetsTraining.ipynb [hunk 1]
```diff
@@ -0,0 +1,611 @@
...
+    "<h1>Working with Jupyter Widgets</h1>\n",
+    "<h3>Unidata AMS 2021 Student Conference</h3>\n",
+    "\n",
+    "<div style=\"clear:both\"></div>\n",
+    "</div>\n",
+    "\n",
+    "<hr style=\"height:2px;\">\n",
+    "\n",
+    "In this notebook, we'll explore several interactive [Jupyter Widgets](https://ipywidgets.readthedocs.io/en/stable/user_guide.html) and use them to incorporate user input to make dynamic, customizable plots.\n",
+    "<div style=\"float:right; width:100 px\"><img src=\"../../instructors/images/widgets_preview.png\" alt=\"example of customizable plot with widgets\" style=\"height: 250px;\"></div>\n",
```
Comment (lesserwhirls) [inline notebooks/notebook_enhancements/NotebookWidgetsTraining.ipynb:23]: It looks like this preview image didn't get added to the PR

**Review by lesserwhirls — CHANGES_REQUESTED** [review]

**Comment by lesserwhirls** [comment]
Looks great! The only issue I see is that the sample image is missing in the PR, so it does not render in the notebook.

File: notebooks/notebook_enhancements/NotebookWidgetsTraining.ipynb (hunk 1 above)
Comment (haileyajohnson) [reply notebooks/notebook_enhancements/NotebookWidgetsTraining.ipynb:23]: =(
Should be fixed now

Comment (lesserwhirls) [reply notebooks/notebook_enhancements/NotebookWidgetsTraining.ipynb:23]: Heck yeah, that image is great!

**Comment by lesserwhirls** [comment]
Awesome - once this goes green I'll merge it in. Thanks @haileyajohnson!

**Review by lesserwhirls — APPROVED** [review]

**Comment by srcarter3** [comment]
I know this notebook has already been merged in, but I was just glancing at it through our website and noticed there is no section break (`---`) and "top" link for Section 4.  @haileyajohnson Do you think you could create another PR with that addition made?

**Comment by haileyajohnson** [comment]
@srcarter3 good catch! done: #84

**Pull request merged**